src/App.js

```diff
@@ -474,6 +488,14 @@ class App extends React.Component {
                   </div>
 
                   <HelpScoutBeacon locator={locator} apps={apps} />
+
+                  <GlobalPreferences
```

Comment:
Perhaps we can put this ahead of the `HelpScoutBeacon`, since the beacon is meant to be above it (we may also want to bump these two component's z-indexes to be one higher to add another explicit" layer in the app).

Reply:
Done. For the `z-index` concern, let's discuss about it and implement it afterwards. I'd like to get this merged now.